generated/nidcpower/nidcpower.proto

```diff
@@ -478,6 +595,46 @@ enum SourceMode {
   SOURCE_MODE_NIDCPOWER_VAL_SEQUENCE = 1021;
 }
 
+message NILCRLoadCompensationSpot {
+  double frequency = 1;
+  sint32 reference_value_type = 2;
+  double reference_value_a = 3;
+  double reference_value_b = 4;
+}
+
+message NILCRMeasurement {
+  double vdc = 1;
+  double idc = 2;
+  double stimulus_frequency = 3;
+  nidevice_grpc.NIComplexNumber ac_voltage = 4;
+  nidevice_grpc.NIComplexNumber ac_current = 5;
+  nidevice_grpc.NIComplexNumber z = 6;
+  double z_magnitude = 7;
+  double z_phase = 8;
+  nidevice_grpc.NIComplexNumber y = 9;
+  double y_magnitude = 10;
+  double y_phase = 11;
+  double ls = 12;
+  double cs = 13;
+  double rs = 14;
+  double lp = 15;
+  double cp = 16;
+  double rp = 17;
+  double d = 18;
+  double q = 19;
+  uint32 measurement_mode = 20;
+  bool dc_in_compliance = 21;
+  bool ac_in_compliance = 22;
+  bool unbalanced = 23;
+  double reserved1 = 24;
```

Comment:
Preferably `reserved` fields should have `include_in_proto=false`. If this is a problem for syncing with `hapigen` that could override the preference. But it doesn't make sense to use this mechanism in the proto api, and it's actually worse if we do start using the reserved fields (since C won't preserve the names, which are meaningful in proto gencode).

It seems like this would be a problem for python too?

Reply:
The python API [does not document the reserved fields](https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/session.py#L4843).

Note also that the following 6 fields we have:
```
  double ls = 12;
  double cs = 13;
  double rs = 14;
  double lp = 15;
  double cp = 16;
  double rp = 17;
```
Correspond to these 2 in python:
```
series_lcr            | LCR                  | The inductance, in henrys, the capacitance, in farads, and the resistance, in ohms....
parallel_lcr          | LCR                  | The inductance, in henrys, the capacitance, in farads, and the resistance, in ohms....
```
I'm wondering whether the more appropriate type here is a nested LCR type with 3 doubles?

Reply:
Yeah, we shouldn't generate these reserved fields. Have a PR up on the hapigen side to strip these out for grpc-device: https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/376469

I updated the imported metadata/proto from a local build of that PR to test it out and it's looking good. Both input and output conversion for the custom_type now just doesn't populate for these reserved fields that were removed in config.py and that all seems to work fine. I will update the imported metadata/proto to an actual `nidcpower_grpc_device` export once that AzDo PR goes in.

Reply:
As for simplifying the types like python, I'm going to avoid that. That'd be extra work to add an LCR message type to the proto and update the generated conversion code to know about it. I'll just leave it as is so it matches the struct in the header (and just doesn't populate the reserved fields).